pubpub/pubpub
# Alphabetize citation list

Addresses #1067
After checking all citation styles we offer, it appears they all alphabetize by the entry itself, and typically that means the author's last name, though if no author is present, the title. Here we alphabetize the list, remove numbers from the citation list, and refactor the handling of citations across the editor.

Screenshot:
![Screenshot from 2021-04-12 17-08-21](https://user-images.githubusercontent.com/14115194/114468839-b69a1500-9bb1-11eb-91f8-627d7935ee4f.png)

Test plan:

+ create [pub with broad variety of citations and footnotes](https://demo.duqduq.org/pub/ple0quof/draft)
  + structured value only footnote
  + unstructured value only footnote
  + both values footnote
  + error-producing structured value footnote
  + structured value only citation
  + unstructured value only citation
  + both values citation
  + error-producing structured value citation
+ ensure alphabetical sort is applied to text of the citation, whether that be the author's last name or the title's first word, as the case may be.
